src/fides/core/user.py

```diff
@@ -139,14 +139,15 @@ def create_command(
         server_url=server_url,
     )
     user_id = user_response.json()["id"]
+    new_user_roles = CONFIG.security.root_user_roles
     update_user_permissions(
         user_id=user_id,
         scopes=CONFIG.security.root_user_scopes,
```

Comment:
I think I missed this in my PR https://github.com/ethyca/fides/pull/2848 to remove user scopes - can you clean up here?  Users can no longer be assigned scopes directly, just roles.  The API ignores the "scopes" key if it is passed in, so this should still all be working, but it would be good to remove this scopes attribute from this function altogether.

Reply:
Oh, thanks for the tip! Will fix :)